src/renderer/components/MainPage.tsx

```diff
@@ -268,12 +269,10 @@ class MainPage extends React.PureComponent<Props, State> {
             });
         });
 
+        window.ipcRenderer.on(APP_MENU_WILL_CLOSE, this.unFocusThreeDotsButton);
```

Comment:
Actually, to my previous comment, this probably will cause issues for keyboard issues, because once they close the menu for whatever reason they will use their focus position. It might not be the worst thing, but I think we should have a quick look to make sure it's not a huge problem for those users.

Reply:
I think in that case if they've pressed `ESC` they would likely lose focus. But any other interaction shouldn't necessarily do that. If that's the only one that's a concern then we're probably okay.